Examples/SnapinsSDKKotlinExample/app/src/javaImplDebug/java/com/salesforce/snapinssdkexample/utils/ServiceSDKUtils.java

```diff
@@ -0,0 +1,173 @@
+                .twoWayVideo(getBooleanPref(context, SosSettingsActivity.KEY_SOS_TWO_WAY_VIDEO_SETTING))
+                .fieldServices(getBooleanPref(context, SosSettingsActivity.KEY_SOS_FS_SETTING))
+                .audio(getBooleanPref(context, SosSettingsActivity.KEY_SOS_AUDIO_SETTING))
+                .build();
+    }
+
+    /**
+     * Helper method to enable SOS polling based on SOS settings or uses sensible default values.
+     */
+    public static void startSosPolling(Context context) {
+        // Start polling for SOS status update (UNKNOWN, AVAILABLE, or UNAVAILABLE)
+        if (!SosAvailability.isPolling()) {
+            SosAvailability.startPolling(context,
+                    getStringPref(context, SosSettingsActivity.KEY_ORG_ID,
+                            context.getString(R.string.pref_sos_org_id_default)),
+                    getStringPref(context, SosSettingsActivity.KEY_DEPLOY_ID,
+                            context.getString(R.string.pref_sos_deployment_id_default)),
+                    getStringPref(context, SosSettingsActivity.KEY_POD_URL,
+                            context.getString(R.string.pref_sos_pod_url_default))
+            );
+        }
+    }
+
+    /**
+     * Creates a case configuration based on case settings or uses sensible default values
+     */
+    public static CaseConfiguration getCaseConfiguration(Context context,
+                                                         CaseClientCallbacks caseClientCallbacks,
+                                                         UserAccount userAccount) {
+        // Create a core configuration instance
+        final CaseConfiguration.Builder caseConfigurationBuilder = new CaseConfiguration.Builder(
+                getStringPref(context, CaseSettingsActivity.KEY_COMMUNITY_URL,
+                        context.getString(R.string.pref_case_community_url_default)),
+                getStringPref(context, CaseSettingsActivity.KEY_CREATE_CASE_ACTION_NAME,
+                        context.getString(R.string.pref_case_action_name_default_no_auth)));
+
+        // Add case client callbacks
+        if (caseClientCallbacks != null)
+            caseConfigurationBuilder.callbacks(caseClientCallbacks);
+
+        // If authenticated configure cases with the user account
+        if (userAccount != null) {
+            caseConfigurationBuilder.caseListName(getStringPref(context,
+                    CaseSettingsActivity.KEY_CASE_LIST_NAME,
+                    context.getString(R.string.pref_case_list_name_default)));
+
+            caseConfigurationBuilder.withAuthConfig(
+                    getAuthTokenProvider(userAccount),
+                    new MobileSdkUser(userAccount.getUserId()));
+
+        }
+
+        return caseConfigurationBuilder.build();
+    }
+
+    /**
+     * Returns a nullable value of the current authenticated user
+     */
+    public static UserAccount getAuthenticatedUser() {
```

Comment:
Maybe change `getAuthenticatedUser` -> `AuthenticatedUser` since it's a `static` accessor? Same on line 163?

Reply:
done